test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py

```diff
@@ -38,7 +38,7 @@
 def test_dgl_gcn_training_cpu(ecr_image, sagemaker_regions, instance_type):
     # TODO: Remove when DGL gpu test on ecs get fixed
     _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
-    if Version(image_framework_version) == Version("1.10"):
+    if Version("1.10") <= Version(image_framework_version) < Version("1.11"):
```

Comment:
Please change this to `Version(image_framework_version) in SpecifierSet("==1.10.*")`
```suggestion
    if Version(image_framework_version) in SpecifierSet("==1.10.*"):
```